app/src/main/java/org/lineageos/glimpse/fragments/MediaViewerFragment.kt

```diff
@@ -152,10 +153,14 @@ class MediaViewerFragment : Fragment(
                     1, 1
                 ),
                 Snackbar.LENGTH_LONG,
-            ).show()
+            ).setAnchorView(bottomSheetLinearLayout).show()
+        }
+    private val favoriteContract =
+        registerForActivityResult(ActivityResultContracts.StartIntentSenderForResult()) {
+            mediaViewerAdapter.getMediaFromMediaStore(viewPager.currentItem)?.let {
```

Comment:
We have to check for its result

Reply:
We just update the icon regardless of the result to reflect the current value, I don't think it should be checked